daffodil-cli/src/main/scala/org/apache/daffodil/Main.scala

```diff
@@ -916,7 +912,10 @@ object Main extends Logging {
         }.map{ _.withExternalVariables(retrieveExternalVariables(parseOpts.vars, cfgFileNode))}
 
         val rc = processor match {
-          case Some(proc) if (!proc.isError) => {
+          case Some(proc) if (proc.isError) => ExitCode.UnableToCreateProcessor
```

Comment:
This case is impossible and can be removed. With the above change, we will never get a Some with isError == true.